### DISCUSSION TITLE
EditForm inside Dialog using DialogService

### DISCUSSION THREAD

**Discussion by acmesoft-arg** [microsoft/fluentui-blazor#1256] [Q&A]
I have a datagrid with a button, in each row, that opens a Dialog of a component using: 
`var d = await DialogService.ShowDialogAsync<MyDialog>(myObject, parameters);`
If I add an EditForm inside the compoment, I want to validate it before close the dialog, I want to use the default buttons of PrimaryAction, and not add others buttons to maintain the design of the dialogs in the app.
There is any way to do this?
I'm using Blazor WebAssembly .net8
Thank you!

**Comment by dvoituron** [comment]
Here you'll find a new example of how to validate an `EditForm` included in a dialog box.

https://github.com/microsoft/fluentui-blazor/pull/1397
